base_location_geonames_import/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
Coding is no longer necessary, as it can be checked in the [migration document to 11.0](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0)

Reply:
But here it still appears:
https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py

Reply:
But here it appears:
https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py

Reply:
Because for Python 2.7 is still needed...

Reply:
Oh, yes!!